x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -695,6 +695,11 @@ public enum Cap {
          */
         JOIN_LOOKUP_SKIP_MV(JOIN_LOOKUP_V11.isEnabled()),
 
+        /**
+         * LOOKUP JOIN without MV matching on lookup index key (https://github.com/elastic/elasticsearch/issues/118780)
+         */
+        JOIN_LOOKUP_SKIP_MV_ON_LOOKUP_KEY(JOIN_LOOKUP_V11.isEnabled()),
```

Comment:
To avoid renaming conficts, I'm adding a new cap. We can remove it later if we want